pgmpy/factors/Factor.py

```diff
@@ -432,10 +433,9 @@ def _str(self, html=False):
                 """<td><b>phi(%s)</b><d></tr>""" % ', '.join(self.variables))
             string_list.append(html_string_header)
         else:
-            string_header = "%s\t\t%s" % ("\t\t".join(self.variables),
-                                          "phi(%s)" % ', '.join(self.variables))
+            string_header = self.scope()
+            string_header.append("phi(%s)" % ','.join(self.variables))
```

Comment:
Use something like this. I

```python
string_header.append("phi({variables})".format(variables=",".join(string_header))
```

It is more explicit then the current formatting method.